### PR TITLE
Chat: remove title and header

### PR DESCRIPTION
**Pull request by Strider2342**

### CODE REVIEW

**Comment by Strider2342** [comment]
[Chat: remove private `title` option and implementation for it](https://trello.com/c/9Mpe0Zyh/9520-chat-remove-private-title-option-and-implementation-for-it)

File: packages/devextreme/js/__internal/ui/chat/chat.ts [hunk 1]
```diff
@@ -32,12 +31,11 @@ const CHAT_CLASS = 'dx-chat';
 const TEXTEDITOR_INPUT_CLASS = 'dx-texteditor-input';
 
 type Properties = ChatProperties & {
-  title: string;
+  dayHeaderFormat?: Format;
```
Comment (EugeniyKiyashko) [inline packages/devextreme/js/__internal/ui/chat/chat.ts]: I don't understand why this change is here. Let's revert this hunk.

File: packages/devextreme/testing/tests/DevExpress.ui.widgets/chatParts/chat.tests.js [hunk 2]
```diff
@@ -108,26 +105,6 @@ QUnit.module('Chat', () => {
         });
     });
 
-    QUnit.module('Header integration', moduleConfig, () => {
```
Comment (EugeniyKiyashko) [inline packages/devextreme/testing/tests/DevExpress.ui.widgets/chatParts/chat.tests.js:116]: You missed the defaultOptions.tests.js file, as well as its dependencies and tests.
```
import ChatHeader from '__internal/ui/chat/header';
```

**Review by EugeniyKiyashko — CHANGES_REQUESTED** [review]

File: packages/devextreme/testing/tests/DevExpress.ui.widgets/chatParts/chat.tests.js (hunk 2 above)
Comment (Strider2342) [reply packages/devextreme/testing/tests/DevExpress.ui.widgets/chatParts/chat.tests.js:116]: 7044d67c2ece3996c9d3e8dbc187cf018682b1b5

File: packages/devextreme/js/__internal/ui/chat/chat.ts (hunk 1 above)
Comment (EugeniyKiyashko) [reply packages/devextreme/js/__internal/ui/chat/chat.ts]: https://github.com/DevExpress/DevExtreme/pull/28327/commits/1ab518ce0c6da4dbd1ea075181642bb35f0dcb41

**Review by EugeniyKiyashko — APPROVED** [review]

**Review by Zedwag — APPROVED** [review]

**Review by marker-dao — APPROVED** [review]
